edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/CloudProxy.cs

```diff
@@ -46,8 +46,9 @@ enum Subscriptions
         readonly ResettableTimer timer;
         readonly AsyncLock timerGuard = new AsyncLock();
         readonly bool closeOnIdleTimeout;
-
+        readonly TimeSpan sdkWaitTimeSeconds = TimeSpan.FromSeconds(50);
```

Comment:
Should we add config for it, instead of hardcoding it?

Reply:
That would make more sense. 50 seconds was just some arbitrary amount of time I came up with for testing. A follow up question I have is would we want as a default time? @vipeller as well